### PR TITLE
Cleanup old script files and packagejson scripts commands

### PR DESCRIPTION
**Pull request by ethanshar**
## Description
Cleanup old script files and packagejson scripts commands

## Changelog
Cleanup old script files and packagejson scripts commands

### CODE REVIEW

File: scripts/release.js [hunk 1]
```diff
@@ -1,77 +0,0 @@
-const cp = require('child_process');
```
Comment (M-i-k-e-l) [inline scripts/release.js:1]: It seems our release file is called `bkRelease`, do we want to change it to `release`?

File: package.json [hunk 2]
```diff
@@ -15,31 +15,22 @@
...
     "test:watch": "jest --watch",
     "lint": "eslint src -c .eslintrc.js --ext .tsx,.ts,.js",
     "lint:fix": "eslint src -c .eslintrc.js --fix",
-    "lint:test": "mocha --compilers js:babel-core/register eslint-rules/tests/lib/rules",
     "xcode": "xed ios",
     "build:dev": "tsc --p tsconfig.dev.json",
-    "build:src": "./node_modules/.bin/babel src --out-dir src --config-file ./src/.babelrc.json --extensions '.ts,.tsx' --ignore 'src/index.ts'",
-    "build:lib": "./node_modules/.bin/babel lib --out-dir lib --config-file ./src/.babelrc.json --extensions '.ts,.tsx'",
-    "build:exports": "npm run build:src && npm run build:lib",
-    "build:packages": "node scripts/buildPackages.js",
```
Comment (M-i-k-e-l) [inline package.json:33]: Isn't `build:packages` useful when adding a new package?

**Review by M-i-k-e-l — COMMENTED** [review]
We can also rename `prRelesaeNotes` to `prReleaseNotes`

File: package.json (hunk 2 above)
Comment (ethanshar) [reply package.json:33]: It's part of `scripts/build` script

File: scripts/release.js (hunk 1 above)
Comment (ethanshar) [reply scripts/release.js:1]: Renamed 👌

File: package.json (hunk 2 above)
Comment (M-i-k-e-l) [reply package.json:33]: I know

File: scripts/release.js (hunk 1 above)
Comment (M-i-k-e-l) [reply scripts/release.js:1]: We'll need to edit the steps once this is merged.

**Review by M-i-k-e-l — APPROVED** [review]
We'll need to edit the steps once this is merged.

**Pull request merged**